ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -678,11 +730,15 @@ Install Kueue Operator Via Cli
     END
 
 Install KServe Dependencies
-    [Documentation]    Install Dependent Operators For KServe
-    Set Suite Variable   ${FILES_RESOURCES_DIRPATH}    tests/Resources/Files
-    Set Suite Variable   ${SUBSCRIPTION_YAML_TEMPLATE_FILEPATH}    ${FILES_RESOURCES_DIRPATH}/isv-operator-subscription.yaml
-    Set Suite Variable   ${OPERATORGROUP_YAML_TEMPLATE_FILEPATH}    ${FILES_RESOURCES_DIRPATH}/isv-operator-group.yaml
-    Install Cert Manager Operator Via Cli
+    [Documentation]    Configure KServe-specific dependencies (operators are installed by Install RHOAI Dependencies)
```

Comment:
I would rename this keyword to better reflect what it does. Something like `Configure KServe`, not install.